test/paper_badge_test_button.html

```diff
@@ -0,0 +1,24 @@
+<link rel="import" href="packages/polymer_interop/polymer.html">
```

Comment:
I have been putting these files under the `fixtures` folder, would you mind moving it there?